### PR TITLE
APPQ-3594 Build Anki using turboscript

### PR DESCRIPTION
**Pull request by huiyu-turbo**
Added a TurboScript for Anki (https://apps.ankiweb.net/)

### CODE REVIEW

**Review by johnsu — CHANGES_REQUESTED** [review]
I ran `turbo build https://raw.githubusercontent.com/turboapps/turbome/ec06748168effff767590467f98745afb8523c85/anki/turbo.me` but failed with setup.exe not found.

```
>turbo build https://raw.githubusercontent.com/turboapps/turbome/ec06748168effff767590467f98745afb8523c85/anki/turbo.me
Using VM 23.6.18 from local
Step 0: meta title="Anki"
Step 1: meta namespace="anki"
Step 2: meta name="anki"
Step 3: using gnu/wget microsoft/powershell
Using image clean:40 from local
Using image wget:1.11.4 from local
Downloading image powershell:5 from https://hub.turbo.net/users/microsoft
Step 4: cmd (mkdir, @SYSDRIVE@\Workspace)

Step 5: workdir @SYSDRIVE@\Workspace
Step 6: batch
        cmd ("echo $webpage = Invoke-WebRequest -Uri "https://apps.ankiweb.net/downloads/archive/" >> GetVersion.ps1")
        cmd ("echo $releases = $webpage.Links.href ^| where {$_ -like "*windows-qt6.exe"} ^| sort >> GetVersion.ps1")
        cmd (echo $latest = $releases[-1] >> GetVersion.ps1)
        cmd ("echo $version = [regex]::Match($latest, "\d+\.\d+\.\d+").Value >> GetVersion.ps1")
        cmd (echo $version >> GetVersion.ps1)

Step 7: cmd ("powershell -executionpolicy remotesigned -File "GetVersion.ps1"")

2.1.65
Step 8: cmd (echo, https://github.com/ankitects/anki/releases/download/2.1.65/anki-2.1.65-windows-qt6.exe)

https://github.com/ankitects/anki/releases/download/2.1.65/anki-2.1.65-windows-qt6.exe
Step 9: cmd (wget, --no-check-certificate, --no-verbose, -O, Setup.exe, %url%)

SYSTEM_WGETRC = c:/progra~1/wget/etc/wgetrc
syswgetrc = c:/progra~1/wget/etc/wgetrc
Step 10: cmd (Setup.exe, /S, /D=c:\Anki)

Step 11: cmd (del, Setup.exe, /Q)

Could Not Find C:\Workspace\Setup.exe
```

File: anki/turbo.me [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+meta title="Anki"
+meta namespace="anki"
+meta name="anki"
+
+
+###################################
+# Pull dependency images
+###################################
+
+using gnu/wget, microsoft/powershell
```
Comment (michaelrush) [inline anki/turbo.me:22]: lets remove the microsoft/powershell image here to keep the build script minimal. This should also fix the build error John and I were seeing on our boxes.

**Review by michaelrush — CHANGES_REQUESTED** [review]

File: anki/turbo.me (hunk 1 above)
Comment (huiyu-turbo) [reply anki/turbo.me]: I removed it and the script can run without PowerShell dependency.

**Comment by michaelrush** [comment]
The build now completes and the application launches but the main window does not render any content. I tried importing a deck from anki.com (File > Import) and studying it (Tools > Study Deck) but nothing appeared.
image

I opened with Studio and enabled Chromium Virtualization and now it works:
image

However I'm not aware of a way to do through through TurboScript. I do see a way to do it through PowerShell though:

**Comment by huiyu-turbo** [comment]
Added `enable ChromiumSupport`. It should work now.

**Review by michaelrush — APPROVED** [review]

**Pull request merged**